### PR TITLE
import ssh_host_ed25519_key

### PR DESCRIPTION
**Pull request by zowoq**

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -184,6 +168,63 @@ def deploy(c, hosts=""):
...
+    Decrypt host private key, install with nixos-anywhere. Use inv install --hosts build01
+    """
+    g = DeployGroup(get_hosts(hosts))
+
+    def anywhere(h: DeployHost) -> None:
+        hostname = h.host.replace(".nix-community.org", "")
+        with tempfile.TemporaryDirectory() as tmpdir:
+            decrypt_host_keys(c, hostname, tmpdir)
+            c.run(
+                f"nix run github:numtide/nixos-anywhere#nixos-anywhere -- --extra-files {tmpdir} --flake .#{hostname} {h.host}"
```
Comment (zowoq) [inline tasks.py:222]: This won't work with aarch64-linux as the default kexec tarball is x86_64.

@Mic92 Could nixos-anywhere be changed to read the system from the flake and set the right tarball?

**Comment by zowoq** [comment]
Moved the tasks.py changes to the disko PR so this just imports the current ssh_host_ed25519_key and prevents rsa keys from being created.

**Review by zimbatm — APPROVED** [review]

File: tasks.py (hunk 1 above)
Comment (Mic92) [reply tasks.py]: Sounds like a good idea until $someone finalizes aarch64 ci.

**Pull request merged**